pkg/kubectl/resource/builder.go

```diff
@@ -595,16 +611,21 @@ func (b *Builder) visitorResult() *Result {
 	}
 
 	if b.selectAll {
-		b.selector = labels.Everything()
+		b.labelSelector = labels.Everything()
 	}
 
 	// visit items specified by paths
 	if len(b.paths) != 0 {
 		return b.visitByPaths()
 	}
 
+	// cannot combine label selector and field selector together
```

Comment:
Why not?

Reply:
Because current `apiserver` does not support this.

Both of `fieldSelector` and `labelSelector` are appended as query params. But they are parsed to different group. It is hard to differentiate them simply, unless we've applied new rules on this.

Reply:
What do you mean, parsed to different group?  If you can't specify field and label selectors at the same time on the server that is a bug.